CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Your new git-extra command should support...
+
+* Should support Mac and Linux(You may need to browse their man page)
```

Comment:
I think we need to be more precise on this one.  We have users on i.e. *BSD and Solaris (i think), and one might forget that those may differ from your conventional Arch or Ubuntu in terms of the shell and maybe even available commands.

I don't actually know how to formulate this in a concise way, but our commands should run on all Ubuntu, Debian, Arch Linux, Linux Mint, Fedora/RedHat, and Bsd's and Solaris. The first few can probably go as Unix-like systems (along with OSX), but although *Bsd is Unix-like, technically, I think it deserves a mention because some of them have more restrictive shells.

I have rewritten the line one too many times now.. 
How about `OSX, Ubuntu, FreeBsd`.

So, if it runs on OSX, no one used Linux-specific stuff like reading from `/proc`, if it runs on Ubuntu it probably runs on most other Linux distros, and if it runs on FreeBsd there isn't any features considered 'unsafe' by FreeBsd and hopefully it runs on OpenBsd and the likes.